pages/linux/nmcli-monitor.md

```diff
@@ -0,0 +1,8 @@
+# nmcli monitor
+
+> Monitor NetworkManager showing changes with connection status and similar.
```

Comment:
Perhaps this will read better?
```suggestion
> Monitor changes to the NetworkManager connection status.
```

Reply:
See what others think, I don't like the `the` and without it, it doesn't work either

Reply:
@SethFalco @navarroaxel

Reply:
I like sbrl suggestion